src/CqrsModule.ts

```diff
@@ -57,4 +61,29 @@ export class CqrsModule implements OnApplicationBootstrap {
         this.eventsBus.publisher = new Publisher(this.eventsBus.subject$, this.moduleRef.get(Producer));
         await this.eventsBus.registerPubsubEvents(this.explorerService.pubsubEvents());
     }
+
+    private static createProviders(): Provider[] {
```

Comment:
It can be moved into the `@Module` decorator (tested for `forRoot`, I have some issues with `forRootAsync`)